lib/util/http/scm-manager.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpBase, type InternalJsonUnsafeOptions } from './http';
+import type { HttpMethod, HttpOptions, HttpResponse } from './types';
+
+let baseUrl: string;
+export const setBaseUrl = (newBaseUrl: string): void => {
+  baseUrl = newBaseUrl.replace(/\/*$/, '/');
```

Comment:
use `ensureTrailingSlash`, there shouldn't be more than one trailing slash, it's a user error otherwise